explorer/testdata/string/empty.carbon

```diff
@@ -0,0 +1,22 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// RUN: %{explorer} %s 2>&1 | \
+// RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes=false %s
+// RUN: %{explorer} --parser_debug --trace_file=- %s 2>&1 | \
+// RUN:   %{FileCheck} --match-full-lines --allow-unused-prefixes %s
+// AUTOUPDATE: %{explorer} %s
```

Comment:
It looks like you're using an old test version. Two things:

1) You can use explorer/lit_autoupdate.py to automatically update a lot of this. (RUN and CHECK lines)

2) This line should just be "AUTOUPDATE"

Notes at https://github.com/carbon-language/carbon-lang/blob/trunk/explorer/README.md#example-programs-regression-tests

Reply:
I synced to the latest version, and this isn't an issue anymore.